libs/designsystem/project.json

```diff
@@ -69,11 +69,28 @@
       "builder": "@storybook/angular:build-storybook",
       "options": {
         "configDir": "libs/designsystem/.storybook",
-        "browserTarget": "designsystem:build",
         "outputDir": "dist/storybook/designsystem",
+        "enableProdMode": false,
```

Comment:
Workaround for [this](https://github.com/storybookjs/storybook/issues/14828#) storybook issue